aerleon/lib/naming.py

```diff
@@ -783,7 +786,7 @@ def ParseYaml(self, file_handle, file_name):
 
         self.ParseDefinitionsObject(file_data, file_name)
 
-    def ParseDefinitionsObject(self, file_data, file_name):
+    def ParseDefinitionsObject(self, file_data: Dict[str, Any], file_name: str) -> None:
```

Comment:
Another "Any" that maybe could be more specific?

Reply:
Done